### PR TITLE
Fixes #93 - Rendered resize handlers in wrapper (optional)

### LINKED ISSUE #93: [Feature] Wrap resize buttons in a div/span
Hi, I would like to rotate the buttons around origin point of the parent div, I've tried to change style for every button with transform, but the result is not working. 

The easiest way is to group the buttons in a div, which would have a single rotate transform. Why? Because I need to change the rotation based on the childs rotation (when you rotate more than 180deg, the drag handlers are opposite, to fix this, I can change the rotation of the buttons, not the content). Example:
```
- resize
-- content - rotate: -130deg
-- handlers - rotate: 49deg
```
This will enable dragging by the bottom to resize box down, not up. Of course, there needs to be a prop to change the style of the div. (the handlers div needs 100% width and height).

Is it possible to add this feature soon ? :) Thanks for the package, works perfectly. For the project I need it I've written rotate button that works perfectly with your package. I could post it later on.

Have a great day,
Martin

### PR DESCRIPTION
**Pull request by pionl**
### Proposed solution
Wrap handler divs in a wrapper if a custom class / style is provided.

### Testing Done
- Added a test case for handlerClass
- Using in production code

Based on #93 issue

### CODE REVIEW

**Comment by bokuweb** [comment]
Thanks for your PR :)
But I still do not understand the issue...
Could you provide your rotated project minimized?

File: src/index.js [hunk 1]
```diff
@@ -397,6 +399,20 @@ export default class Resizable extends Component {
       }
       return null;
     });
+
+    const wrapperClass = handlerClasses && handlerClasses['wrapper']
+    const wrapperStyle = handlerStyles && handlerStyles['wrapper']
+
+      // #93 Wrap the resize box in span (will not break 100% width/height)
+    if (wrapperClass || wrapperStyle) {
+      return <span
```
Comment (bokuweb) [inline src/index.js]: Sorry, I still can not understand the need for this wrapper

File: README.md [hunk 2]
```diff
@@ -110,7 +110,7 @@ but other handler will still use the default styles.
 
 #### `handlerClasses?: HandlersClassName;`
 
-The `handlerClasses` property is used to set the className of one or more resize handlers.
+The `handlerClasses` property is used to set the className of one or more resize handlers. You can set className for span wrapper with `wrapper` index.
```
Comment (bokuweb) [inline README.md]: I am at a loss as to whether it is better to separate props.

**Comment by pionl** [comment]
Unfortunately I'm unable to show you the code, but made GIF with real example. With wrapped span/div I can rotate the bars and still support resizing from sides as user expects. When I rotate whole `resizable` div the resize logic is opposite (for user un-logical). If you watch closely, you can see that I'm setting different rotate values for resize wrapper (also width/height to enable portrait resize).

DEMO GIF

Best regards,
Martin

**Comment by bokuweb** [comment]
@pionl Thanks for your reply 😄  Unfortunately I can not see the GIF image. Could you please re-link it??

**Comment by pionl** [comment]
Ok, github was unabel to fetch the gif. I've updated the comment and also inserting it [here](http://g.recordit.co/eOPOFSdizD.gif)

**Comment by bokuweb** [comment]
Your explanation really helped!

File: README.md (hunk 2 above)
Comment (bokuweb) [inline README.md]: @pionl Could you please separate props, like `handlerWrapperClass`, `handlerWrapperStyle`. They are optional not to break, for now. However, I want to set these props and wrapper default.

**Comment by pionl** [comment]
HI @bokuweb , sorry I was out I missed the comment. I see you already made the change. Thank you!

**Comment by bokuweb** [comment]
@pionl Hi! I've merged this PR. Then I'm gonna release this change as v3.0.

https://github.com/bokuweb/react-resizable-box/tree/3.0

you can try it  by `npm i -S re-resizable` for now. (I'm gonna rename this package to `re-resizable` ).
Resizers wrapper is default on v3.0.0. you can pass `handleWrapperStyle` and `handleWrapperClass` props for resizers wrapper. 

thanks.

**Pull request merged**